routers/user.py

```diff
@@ -0,0 +1,259 @@
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+    ]
+    return {"resources": resources}
+
+
+@router.get("/me/resources/approved")
+async def get_approved_resources(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    approved_resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+        if resource.get("status") == "approved"
+    ]
+    return {"approved_resources": approved_resources}
+
+
+@router.get("/me/resources/pending")
+async def get_pending_resources(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    pending_resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+        if resource.get("status") == "pending"
+    ]
+    return {"pending_resources": pending_resources}
+
+
+@router.get("/me/all/pending")
+async def get_all_pending(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    pending_services = [
+        service for service in services
+        if service.get("status") == "pending"
+    ]
+    pending_resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+        if resource.get("status") == "pending"
+    ]
+    return {
+        "pending_services": pending_services,
+        "pending_resources": pending_resources,
+    }
+
+
+@router.post("/request/service")
+async def request_service(
+    service_request: ServiceRequest,
+    user: User = Depends(get_current_user)
+):
+    """Submit a request for a service."""
+    if user.access_token.sub != service_request.user_id:
+        raise HTTPException(
+            status_code=403,
+            detail="User ID in request does not match authenticated user"
+        )
+
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+
+    if any(s.get("id") == service_request.id for s in services):
+        raise HTTPException(
+            status_code=400,
+            detail=f"Service request with ID {service_request.id} already exists"
+        )
+
+    new_service = {
```

Comment:
We should have a schema/model for this so we know what the data should look like.